cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/parser.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export interface ITableMetricsData {
+  tableName: string;
+  errors: number;
+  eventsPerMin: number;
+  latency: number;
+}
+
+interface IMetricData {
```

Comment:
Are any of these types shareable between the charting parsers? Is there a common data model that the back end is using?

Reply:
yes, the metrics data is sharable. I will refactor this into a common place, but limiting the conflict for now until the previous PRs are merged